Sources/SwiftSyntaxBuilder/gyb_generated/DeclBuildables.swift

```diff
@@ -0,0 +1,1082 @@
+    trailingComma: TokenSyntax? = Tokens.`comma`
+  ) {
+    self.init(
+      specifier: specifierBuilder(),
+      name: SyntaxFactory.makeIdentifier(name),
+      assignToken: assignToken,
+      expression: expression,
+      trailingComma: trailingComma
+    )
+  }
+}
+
+extension ClosureCaptureSignature {
+  public init(
+    leftSquare: TokenSyntax = Tokens.`leftSquareBracket`,
+    @ClosureCaptureItemListBuilder itemsBuilder: () -> ClosureCaptureItemList = { .empty },
+    rightSquare: TokenSyntax = Tokens.`rightSquareBracket`
+  ) {
+    self.init(
+      leftSquare: leftSquare,
+      items: itemsBuilder(),
+      rightSquare: rightSquare
+    )
+  }
+}
+
+extension ClosureExpr {
+  public init(
+    leftBrace: TokenSyntax = Tokens.`leftBrace`,
+    signature: ClosureSignature? = nil,
+    @CodeBlockItemListBuilder statementsBuilder: () -> CodeBlockItemList = { .empty },
+    rightBrace: TokenSyntax = Tokens.`rightBrace`
+  ) {
+    self.init(
+      leftBrace: leftBrace,
+      signature: signature,
+      statements: statementsBuilder(),
+      rightBrace: rightBrace
+    )
+  }
+}
+
+extension FunctionCallExpr {
+  public init(
+    calledExpression: ExprBuildable,
+    leftParen: TokenSyntax? = Tokens.`leftParen`,
+    @TupleExprElementListBuilder argumentListBuilder: () -> TupleExprElementList = { .empty },
+    rightParen: TokenSyntax? = Tokens.`rightParen`,
+    trailingClosure: ClosureExpr? = nil,
+    @MultipleTrailingClosureElementListBuilder additionalTrailingClosuresBuilder: () -> MultipleTrailingClosureElementList = { .empty }
+  ) {
+    self.init(
+      calledExpression: calledExpression,
+      leftParen: leftParen,
+      argumentList: argumentListBuilder(),
+      rightParen: rightParen,
+      trailingClosure: trailingClosure,
+      additionalTrailingClosures: additionalTrailingClosuresBuilder()
+    )
+  }
+}
+
+extension SubscriptExpr {
+  public init(
+    calledExpression: ExprBuildable,
+    leftBracket: TokenSyntax = Tokens.`leftSquareBracket`,
+    @TupleExprElementListBuilder argumentListBuilder: () -> TupleExprElementList = { .empty },
+    rightBracket: TokenSyntax = Tokens.`rightSquareBracket`,
+    trailingClosure: ClosureExpr? = nil,
+    @MultipleTrailingClosureElementListBuilder additionalTrailingClosuresBuilder: () -> MultipleTrailingClosureElementList = { .empty }
+  ) {
+    self.init(
+      calledExpression: calledExpression,
+      leftBracket: leftBracket,
+      argumentList: argumentListBuilder(),
+      rightBracket: rightBracket,
+      trailingClosure: trailingClosure,
+      additionalTrailingClosures: additionalTrailingClosuresBuilder()
+    )
+  }
+}
+
+extension ExpressionSegment {
+  public init(
+    backslash: TokenSyntax = Tokens.`backslash`,
+    delimiter: String?,
+    leftParen: TokenSyntax = Tokens.`leftParen`,
+    @TupleExprElementListBuilder expressionsBuilder: () -> TupleExprElementList = { .empty },
+    rightParen: TokenSyntax = Tokens.`stringInterpolationAnchor`
```

Comment:
There is something wrong here. 
I think it relates to this [ExpressionSegment](https://github.com/apple/swift/blob/main/utils/gyb_syntax_support/ExprNodes.py#L504)

On line 510 left paren is `  Child('LeftParen', kind='LeftParenToken', classification='StringInterpolationAnchor', force_classification=True),`

But the right paren `Child('RightParen', kind='StringInterpolationAnchorToken')`

I think it is a bug?